operator/pkg/controller/istiocontrolplane/istiocontrolplane_controller.go

```diff
@@ -230,16 +230,12 @@ func (r *ReconcileIstioOperator) Reconcile(_ context.Context, request reconcile.
 		if ir := iop.Annotations[IgnoreReconcileAnnotation]; ir == "true" {
 			scope.Infof("Ignoring the IstioOperator CR %s because it is annotated to be ignored for reconcile ", iopName)
 			return reconcile.Result{}, nil
+		} else if is := iop.Annotations[name.InstalledSpecCRPrefix]; is != "" {
+			scope.Infof("Ignoring the installed-state IstioOperator CR %s ", iopName)
+			return reconcile.Result{}, nil
```

Comment:
Does the saved IOP has this annotation. IIRC, it does have ignore reconcile annotation.